src/lib/fonts-css.ts

```diff
@@ -16,7 +17,7 @@ interface FontDisplay {
     woff2: string;
     woff: string;
     ttf: string;
-    weight: number;
+    weight: string | number;
```

Comment:
if `weight` can be `string` or `number` would you not need to cast `weight` to a `string` to get conditions like `font.weight === '400'` to work?